### DISCUSSION TITLE
DDA Port: Mutation Selector mod.

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#3085] [General]
https://github.com/CleverRaven/Cataclysm-DDA/pull/67483

I skimmed it over to see if its EOC reliant and I don't think it is, but I could be wrong. Would be a nice addition, maybe the original author would see this but they haven't been active on Github in a while.

**Comment by scarf005** [comment]
yes, this PR does not use EOC. would be relatively straightforward to port.
